src/main/java/org/assertj/core/util/Strings.java

```diff
@@ -83,7 +83,8 @@ public static String concat(Object... objects) {
    * @return the formatted string if any args were given
    */
   public static String formatIfArgs(String message, Object... args) {
-    return Arrays.isNullOrEmpty(args) ? message : format(message, args);
+    return Arrays.isNullOrEmpty(args) ? format(message.replaceAll("%","%%").replaceAll("%%n","%n")).intern() : format
```

Comment:
That's exactly the reason but I didn't want to change the tests while changing production code and without talking to you about it first. Now I'll do :D
I agree it do need a comment but I rather find a more elegent maner…